t/access_log.t

```diff
@@ -17,7 +17,11 @@ use Test::More;
 my ($app, $jmap_tester) = Bakesale::Test->new_test_app_and_tester;
 \my %account = Bakesale::Test->load_trivial_account($app->processor->schema_connection);
 
-$jmap_tester->_set_cookie('bakesaleUserId', $account{users}{rjbs});
```

Comment:
As part of my "why not make upgrading as easy as possible", it should be easy to shim this in JMAP::Tester so that `_set_cookie` continued to do the right thing, so that no change would be needed here.